### PR TITLE
#6 - Added basic PR template

### LINKED ISSUE #6: Pull Request Template
Add things like DoD, tests, linting, formatting, etc, etc, etc
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository

### PR DESCRIPTION
**Pull request by kcarlson**
Closes #6 

Basic and simple PR template unabashedly stolen from parts of the internet.
Open for suggestions please!

Guide: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Describe your changes
+
+## Issue ticket number and link
+
+## Checklist before requesting a review
+- [ ] I have performed a self-review of my code
+- [ ] If it is a core feature, I have added thorough tests.
+
```
Comment (wooyey) [inline pull_request_template.md:9]: Maybe?:

- [x] Documentation was updated/added (if needed)

**Review by wooyey — APPROVED** [review]
Almost there ;)

File: pull_request_template.md (hunk 1 above)
Comment (kcarlson) [reply pull_request_template.md:9]: :heavy_check_mark:

**Review by tjonsson — APPROVED** [review]

**Review by eharhad — COMMENTED** [review]
:-) - OK!

**Pull request merged**